test/parallel/test-repl-eval.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const repl = require('repl');
+
+{
```

Comment:
It's there so that if we write a subsequent test (such as to test that `context` is sent on tab completion), we can make sure there are no side effects (because the variables are scoped to the block).